models/music_album.rb

```diff
@@ -0,0 +1,10 @@
+require './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date, id = Random.rand(1...1000))
+    super(id, publish_date)
+    @on_spotify = on_spotify
+  end
```

Comment:
It would be great if the id value, Random was put in the body of the constructor and not as an argument.

Reply:
Resolved, thanks.

Reply:
Great for implimenting those changes :+1: :partying_face: :rocket: